pkg/engine/engine.go

```diff
@@ -23,6 +23,7 @@ var AllBuilders = []build.Builder{
 var AllRunners = []runner.Runner{
 	&runner.LocalDockerRunner{},
 	&runner.LocalExecutableRunner{},
+	&runner.ClusterSwarmRunner{},
```

Comment:
```suggestion
	&runner.ClusterDockerSwarmRunner{},
```

Reply:
I'm not entirely sure if naming yet another Runner just for Docker Swarm is the best path. It probably worth to just have Docker Swarm as the default Docker Runner

Reply:
Or perhaps this is me just getting confused by seeing Docker Swarm being called a Runner (given that in the end it is just the same thing running in a Docker container, but managed by Docker Swarm)

Reply:
I think calling this `ClusterSwarmRunner` is fine. I was using the convention `scope:technology`, e.g. `local:exec`, `local:docker`, etc. to name runners. We can also rename "runners" to "schedulers" if we feel that's more accurate.